data/test/demo-rasa-composite-entities.md

```diff
@@ -0,0 +1,75 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+  - [intent:affirm](#intentaffirm)
+  - [intent:goodbye](#intentgoodbye)
+  - [intent:greet](#intentgreet)
+  - [intent:chitchat](#intentchitchat)
+  - [intent:restaurant_search](#intentrestaurant_search)
+- [intent:order_pizza](#intentorder_pizza)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## intent:affirm
+- yes
+- yep
+- yeah
+- indeed
+- that's right
+- ok
+- great
+- right, thank you
+- correct
+- great choice
+- sounds really good
+
+## intent:goodbye
+- bye
+- goodbye
+- good bye
+- stop
+- end
+- farewell
+- Bye bye
+- have a good one
+
+## intent:greet
+- hey
+- howdy
+- hey there
+- hello
+- hi
+- good morning
+- good evening
+- dear sir
+
+## intent:chitchat
+- What's your name?
+- What can I call you?
+- How's the weather?
+- Is it too hot outside?
+
+## intent:restaurant_search
+- i'm looking for a place to eat
+- I want to grab lunch
+- I am searching for a dinner spot
+- i'm looking for a place in the [north](location) of town
```

Comment:
is it equivalent to `i'm looking for a place in the [north]{"entity": "location"} of town`
I'm afraid different type of parenthesis will be confusing for users

Reply:
We support both. In a standard case the user just needs `[north](location)` which is more readable. We discussed that and agreed that we are going to keep this simple format.

Reply:
I meant shouldn't we use `(` instead of `{`?

Reply:
Not sure if that is more intuitive as `{}` is used for a dict in python, which we are basically using here. But open for a discussion. However, I would not change the data format in this PR, if we decide to change it again, would create a new PR.

Reply:
I just thought that users who create train data might not be familiar with any programming language. Any harm support both?

Reply:
Would not support both, might be confusing for the user. Let's move the discussion to Slack.